frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/resource-utils.ts

```diff
@@ -69,7 +69,7 @@ export const createEventListener = async (
       triggers: [
         {
           bindings: triggerBindings.map(mapTriggerBindings),
-          template: { name: triggerTemplate.metadata.name },
+          template: { ref: triggerTemplate.metadata.name },
```

Comment:
What version was it added in? Do we have any concerns with a 4.7 cluster having like a 4.6 operator install version?

Reply:
I checked the version. It was added in 0.9.x so ocp-4.6 and below wont have this change. I will conditionally add ref based on the operator version.

Reply:
Updated based on the operator version.